### PR TITLE
Pkg

### PR DESCRIPTION
**Pull request by xcko**
misc. pkgs
update ca-certificates
add libgit2
add stagit
add stagit-gopher
add geomyidae

### CODE REVIEW

File: pkg/ca-certificates [hunk 1]
```diff
@@ -48,6 +48,7 @@ cp sbin/update-ca-certificates .
 sed -e 's#=/etc/#=${DESTDIR}/etc/#' -i update-ca-certificates
 sed -e 's#=/share/#=${DESTDIR}/share/#' -i update-ca-certificates
 sed -e 's#=/local/#=${DESTDIR}/local/#' -i update-ca-certificates
+sed -e 's#openssl rehash#c_rehash#' -i update-ca-certificates
```
Comment (rofl0r) [inline pkg/ca-certificates]: c_rehash tool is coming from which pkg ? it's not in my $PATH

**Comment by rofl0r** [comment]
i'm not very delighted about the -git packages. we can add them, but it'd be highly preferable if there's a way to download a tarball from a tag (most git-web packages allow that in some form)
example: pkg/john

**Comment by xcko** [comment]
I can remove the git versions of stagit and stagit-gopher if you want

**Comment by rofl0r** [comment]
no it's OK, i thought their git-web interface may supply a possibility to create a tarball from a commit hash.
the author may also respond to a request...

File: pkg/ca-certificates (hunk 1 above)
Comment (rofl0r) [reply pkg/ca-certificates]: alright, couldn't remember all the details! :+1:

**Comment by xcko** [comment]
okay I'll update if I get a response.

**Comment by xcko** [comment]
found an ftp source. not ideal, but better than git

**Comment by rofl0r** [comment]
cool. are those versions sufficiently modern that we can remove the -git packages ?

**Comment by xcko** [comment]
yup. I'll chuck them.

File: pkg/libgit2 [hunk 2]
```diff
@@ -0,0 +1,41 @@
+[mirrors]
+https://github.com/libgit2/libgit2/archive/v0.27.4.tar.gz
+
+[vars]
+filesize=4772254
+sha512=d27db86eb1b9f0d4057f8538ba1985ee76c3ca106e57d417fa9bff79d575f91a07ad28693112b58dc1d61d68116a82e6a145f12276158f2806b6c4964d741f61
+tardir=libgit2-0.27.4
```
Comment (rofl0r) [inline pkg/libgit2]: could you add a tarball directive here, to prevent the tarball from being saved as v0.27.tar.gz

see pkg/john as example

Comment (xcko) [reply pkg/libgit2]: ok

**Pull request merged**